sdk/src/main/java/com/qonversion/android/sdk/internal/services/QFallbacksService.kt

```diff
@@ -17,10 +17,11 @@ internal class QFallbacksService(
     private val logger: Logger
 ) {
     private val jsonAdapter: JsonAdapter<QFallbackObject> = moshi.adapter(QFallbackObject::class.java)
+    private val filePath = "qonversion_fallbacks.json"
```

Comment:
It's local variable now, to make it constant move above the class name and rewrite as follows:
```
private const val FILE_PATH = "qonversion_fallbacks.json"
```

And let's name it FALLBACK_FILE_NAME? as it's not path?